2.1.7/alpine3.11/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN set -eux; \
 	apk del --no-cache --no-network buildtmp
 
 COPY nats-server.conf /etc/nats/nats-server.conf
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
I noticed the Postgres Dockerfile puts `docker-entrypoint.sh` in `/usr/local/bin/`.
https://github.com/docker-library/postgres/blob/682ff83c5c83f1b6f2b02caf7aa3e17a491b403a/13/Dockerfile#L185

Totally not important or necessary, but it cleans up the `/` directory a little.